packages/amplify-provider-awscloudformation/lib/initializer.js

```diff
@@ -58,21 +58,18 @@ async function run(context) {
 
 async function getConfiguredAwsCfnClient(context) {
```

Comment:
If this function does not return the configured aws client anymore, but only the aws config, its name should be appropriately updated to reflect its functionality.